src/site/content/en/blog/web-animations/index.md

```diff
@@ -0,0 +1,262 @@
+---
+title: "What's coming to Chrome 84 in the Web Animations API"
+subhead: Wrangling your web animations is about to get much easier
+authors:
+  - una
+  - kevinellis
+date: 2020-05-27
+hero: hero.png
+# You can adjust the position of your hero image with this property.
+# Values: top | bottom | center (default)
+# hero_position: bottom
+# thumbnail: thumbnail.png
+description: Wrangling your web animations is about to get much easier.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - web animations
+  - waapi
+  - chrome84
+---
+
+When used correctly, animations improve user perception and [memory](https://www.researchgate.net/publication/229351931_The_Effects_of_Animation_and_Format_on_the_Perception_and_Memory_of_Online_Advertising) of your brand, guide user actions, and help users navigate your application—providing context in a digital space.
+
+<figure class="w-figure">
+  <img class="w-screenshot" src='./waapi-timeline.png' alt="The Web Animations API first hit Chrome in version 36, July of 2014. Now the spec is going to be complete, in version 84, launching July 2020.">
+  <figcaption class="w-figcaption">
+    The long history of the Web Animations API.
+  </figcaption>
+</figure>
+
+The [Web Animations API](https://www.w3.org/TR/web-animations-1/) is a tool that enables developers to write [imperative](https://www.youtube.com/watch?v=WaNoqBAp8NI) animations with JavaScript. It was written to underpin both CSS animation and transition implementations and enable future effects to be developed, as well as existing effects to be composed and timed.
+
+While it has a long history, originally launching in Chrome 36, the latest release, Chrome 84, brings us a slew of previously unsupported features. 
+
+## Getting started
+
+Creating an animation via the web animations API should feel very familiar if you've used `"@keyframe"` rules. First you'll need to create a Keyframe Object. What might look like [this](https://codepen.io/una/pen/RwWMvPw) in CSS:
```

Comment:
Just to clarify, my request for quotes around keywords was only for the names of the modes because the quotes are part of what needs to be in code. If I led you to overcorrect, I apologize.

+1 on the link. Try to link on text that describes what's at the end of the link.